doc/user-guides/acrn-shell.rst

```diff
@@ -44,3 +44,9 @@ The ACRN hypervisor shell supports the following commands:
      - Display the CPUID leaf [subleaf], in hexadecimal
    * - reboot
      - Trigger a system reboot (immediately)
+   * - rdmsr [-p<pcpu_id>] <msr_index>
+     - Read the Model-Specific Register (MSR) at index ``msr_index`` (in hex)
```

Comment:
better would be to use "hexadecimal" as used above (fix other uses of "hex" too)

Reply:
> better would be to use "hexadecimal" as used above (fix other uses of "hex" too)

I agree... done